filters/PMFFilter.cpp

```diff
@@ -105,7 +105,11 @@ std::vector<double> PMFFilter::morphOpen(PointViewPtr view, float radius)
     // dilate
     for (PointId i = 0; i < np; ++i)
```

Comment:
Confused by this.  Why do you go through the points twice?  Can't you compute both the local min and local max on one pass through the loop and one fetch of the neighbors?  Maybe I'm missing something.